submissions/lalalilulu/frogger-game/app.js

```diff
@@ -0,0 +1,112 @@
+const dx = 101;
+const dy = 80;
+const collision_offset = 60;
+const player_start_x = dx * 2;
+const player_start_y = dy * 5;
+
+function calculateRandomSpeed() {
+    return Math.random() * (300 - 50) + 50;
+}
+
+class Character {
+
+    constructor(x, y, image) {
+        this.x = x;
+        this.y = y;
+        this.image = image;
+    }
+
+    render() {
+        ctx.drawImage(Resources.get(this.image), this.x, this.y);
+    }
+}
+
+
+class Enemy extends Character {
+
+    constructor(x, y, player, speed, image = 'images/enemy-bug.png') {
```

Comment:
IDE must have warned you that the `speed` argument is not used.

Reply:
I have removed the speed variable from the constructor